cloudformation.yaml

```diff
@@ -222,15 +339,12 @@ Resources:
   LambdaFunc:
     Type: 'AWS::Lambda::Function'
     Properties:
-      Code:
-        S3Bucket: !Ref S3Bucket
-        S3Key: !Ref CodeKey
-      Handler: example.EntryPoint
+      %LAMBDACONFIG%
```

Comment:
I suspect the same thing, that this is just a comment since there is no Lambda config needed